src/js/entities-service/entities/groups.js

```diff
@@ -11,6 +13,51 @@ const _Model = BaseModel.extend({
   getClinicians() {
     return Radio.request('entities', 'clinicians:collection', this.get('_clinicians'));
   },
+  addClinician(clinician) {
+    const url = `/api/groups/${ this.id }/relationships/clinicians`;
+    const groups = clinician.get('_groups') || [];
+
+    clinician.set({ _groups: groups.concat({ id: this.id }) });
+
+    this.set({ _clinicians: this.get('_clinicians').concat({
+      id: clinician.id,
+      type: clinician.type,
```

Comment:
only needs to be id on the `_foo` relationships.  I think adding type would actually end up adding type to the data.